corehq/sql_db/routers.py

```diff
@@ -87,7 +87,7 @@ def allow_migrate(db, app_label, model_name=None):
     elif app_label == SQL_ACCESSORS_APP:
         return db in partition_config.get_form_processing_dbs()
     else:
-        return db == partition_config.get_main_db()
+        return db == 'default'
```

Comment:
Does Django have a named constant for this? If not, it might be worth us defining one.

Reply:
There is actually `django.db.DEFAULT_DB_ALIAS`. I'll do that in a separate PR

Reply:
https://github.com/dimagi/commcare-hq/pull/25691